datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -288,6 +287,25 @@ def create_scraper_configuration(self, instance=None):
         if config['metadata_metric_name'] and config['metadata_label_map']:
             config['_default_metric_transformers'][config['metadata_metric_name']] = self.transform_metadata
 
+        # Set up the HTTP wrapper for this endpoint
+
+        # TODO: Deprecate this behavior
+        if config['ssl_verify'] is False:
+            config.setdefault('tls_ignore_warning', True)
+
+        http_handler = self.http_handlers[endpoint] = RequestsWrapper(
+            config, self.init_config, self.HTTP_CONFIG_REMAPPER, self.log
+        )
+
+        headers = http_handler.options['headers']
+
+        bearer_token = config['_bearer_token']
+        if bearer_token is not None:
+            headers['Authorization'] = 'Bearer {}'.format(bearer_token)
```

Comment:
I know it was already in the old code, but is there a test for it? If not, maybe we should add one.

Reply:
We can in subsequent PRs, I'm trying to keep the diff small